pkg/controller/operators/olm/operator.go

```diff
@@ -537,9 +541,17 @@ func (a *Operator) operatorGroupForActiveCSV(logger *logrus.Entry, csv *v1alpha1
 		return nil
 	}
 
+	targets, ok := annotations[v1alpha2.OperatorGroupTargetsAnnotationKey]
+
+	// No target annotation
+	if !ok {
+		logger.Info("no olm.targetNamespaces annotation")
+		return nil
```

Comment:
Before adding the return, we would always associate a missing `olm.targetNamespaces` annotation with `NamespaceAll`.